src/test/resources/features/refapp-2.x/14-clinicalVisit/clinicalVisit.feature

```diff
@@ -50,3 +50,55 @@ Feature: Clinical Visit Management
     When a user attaches patient supporting file
     And a user clicks the upload file button
     Then the system uploads the file
+
+  @selenium
+  @clinicalVisit
+  Scenario: Patient Dashboard adding Allergy
+    # User story: Adding New Allergy
+    When user clicks on Allergies link from Patient dashboard
+    Then system loads Allergies table page
+    When a user clicks Add New Allergy button
+    Then the system loads Add New Allergy dashboard
+    And the user selects a new Allergy
+    And the user clicks on the Save button
+    Then the system adds a new Allergy
+
+  @selenium
+  @clinicalVisit
+  Scenario: Patient Dashboard adding condition
+    # User story: Add New Condition
+    When a user clicks on Conditions on Patient dashboard
+    Then system loads Conditions Page
+    When user clicks on Add new condition button
+    And user enters patient condition
+    And user clicks on save button
+    Then system adds New Condition in Conditions table
+
+  @selenium
+  @clinicalVisit
+  Scenario: Patient Dashboard adding Allergy2
```

Comment:
@Ndacyayisenga-droid did you notice in the feature [file](https://github.com/openmrs/openmrs-contrib-qaframework/blob/f131c91a62c19fdad0c3742d55cc9dbae1aad9e4/src/test/resources/features/refapp-2.x/14-clinicalVisit/clinicalVisit.feature#L18-L23) there is one allergy added already, why not now adding only one more?

Reply:
> @Ndacyayisenga-droid did you notice in the feature [file](https://github.com/openmrs/openmrs-contrib-qaframework/blob/f131c91a62c19fdad0c3742d55cc9dbae1aad9e4/src/test/resources/features/refapp-2.x/14-clinicalVisit/clinicalVisit.feature#L18-L23) there is one allergy added already, why not now adding only one more?

@kdaud is there a difference between `known allergy` and `new allergy` ?

Reply:
@Ndacyayisenga-droid  what do you think is the difference?

Reply:
> @Ndacyayisenga-droid what do you think is the difference?

I reaaly dont know the difference but you can shade more light
![show](https://user-images.githubusercontent.com/58124613/135339609-f9ce29cd-d534-412a-bfb4-fd3089cc7072.png)
.

Reply:
@Ndacyayisenga-droid have you played around with those two buttons and realized what each is  accomplishing?

BTW do you understand what an allergy is all about?

Reply:
@Ndacyayisenga-droid did you see the reviews above?